### PR TITLE
Updates to third party scripts

### PR DESCRIPTION
**Pull request by blowekamp**
Remove legacy UpdateThirdParyFromUpstream script

Fix pre-commit in subtree missing configuration file.

## PR Checklist
- [ ] No [API changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#breaking-changes) were made (or the changes have been approved)
- [ ] No [major design changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#design-changes) were made (or the changes have been approved)
- [ ] Added test (or behavior not changed)
- [ ] Updated API documentation (or API not changed)
- [ ] Added [license](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Utilities/KWStyle/ITKHeader.h) to new files (if any)
- [ ] Added Python wrapping to new files (if any) as described in [ITK Software Guide](https://itk.org/ItkSoftwareGuide.pdf) Section 9.5
- [ ] Added [ITK examples](https://github.com/InsightSoftwareConsortium/ITKSphinxExamples) for all new major features (if any)

Refer to the [ITK Software Guide](https://itk.org/ItkSoftwareGuide.pdf) for
further development details if necessary.

### CODE REVIEW

File: Modules/ThirdParty/PNG/UpdateFromUpstream.sh
Comment (dzenanz) [inline Modules/ThirdParty/PNG/UpdateFromUpstream.sh:29]: Does this work?

**Review by dzenanz — APPROVED** [review]

**Review by bradking — APPROVED** [review]

File: Modules/ThirdParty/PNG/UpdateFromUpstream.sh
Comment (blowekamp) [reply Modules/ThirdParty/PNG/UpdateFromUpstream.sh]: After a proper update it still doesn't work. There is the follow errors during the script:
```
+ a1d11567ee...47ed77d879 upstream-itkpng -> upstream-itkpng  (forced update)
++ git merge --log -s ours --no-commit --allow-unrelated-histories upstream-itkpng
Automatic merge went well; stopped before committing as requested
++ git read-tree -u --prefix=Modules/ThirdParty/PNG/src/itkpng/ upstream-itkpng
error: Entry 'Modules/ThirdParty/PNG/src/itkpng/LICENSE' overlaps with 'Modules/ThirdParty/PNG/src/itkpng/LICENSE'.  Cannot bind.
+ rm -rf /Users/blowekamp/src/ITK/work
```

Comment (bradking) [reply Modules/ThirdParty/PNG/UpdateFromUpstream.sh]: We may need a one-time manual re-import of the existing PNG version with the new script.  I'll look at this and push an update.

Comment (blowekamp) [reply Modules/ThirdParty/PNG/UpdateFromUpstream.sh]: Maybe cherry-pick 7abd4e0db581e37d22fca052323d45088144078e and make another PR.

**Review by thewtex — APPROVED** [review]

File: Modules/ThirdParty/PNG/UpdateFromUpstream.sh
Comment (bradking) [reply Modules/ThirdParty/PNG/UpdateFromUpstream.sh]: #5054 re-imports PNG 1.6.44 with the new import script.  Once that is merged, this PR can be rebased.

Comment (blowekamp) [reply Modules/ThirdParty/PNG/UpdateFromUpstream.sh]: @hjmjohnson Thank your work on ITK, but please be more careful on the merge button. see the above message.

Comment (bradking) [reply Modules/ThirdParty/PNG/UpdateFromUpstream.sh]: We should have revoked our approvals during the review process.

**Pull request merged**
